packages/devtools_app/lib/src/app.dart

```diff
@@ -613,15 +625,10 @@ List<DevToolsScreen> get defaultScreens {
       VMDeveloperToolsScreen(controller: vmDeveloperToolsController),
       controller: vmDeveloperToolsController,
     ),
-    DevToolsScreen<void>(
-      const ProviderScreen(),
-      createController: () {},
-    ),
-// Uncomment to see a sample implementation of a conditional screen.
-//      DevToolsScreen<ExampleController>(
-//        const ExampleConditionalScreen(),
-//        createController: () => ExampleController(),
-//        supportsOffline: true,
-//      ),
   ];
+
+  // Show the sample DevTools screen.
+  if (enableSampleScreen) screen.add(sampleConditionScreen);
```

Comment:
only ever include this screen in debug + profile builds.

Reply:
Good idea - Done.